corgidrp/l3_to_l4.py

```diff
@@ -1679,9 +1698,13 @@ def subtract_stellar_polarization(input_dataset, system_mueller_matrix_cal, nd_m
             # propagate errors for the subtraction
             sum_err = np.sqrt(frame.err[0,0,:,:]**2 + frame.err[0,1,:,:]**2)
             diff_err = sum_err
-            diff_err = np.sqrt(diff_err**2 + 
-                               (sum * normalized_diff * np.sqrt((sum_err/sum)**2 + (normalized_diff_err/normalized_diff)**2))**2
-                        )
+            #Ignore RuntimeWarnings here because of the potential to divide by zero (especially in tests). 
+            with warnings.catch_warnings():
```

Comment:
I think Juergen is deleting the 2nd copy of subtract stellar polarization, so maybe also do that, and make the changes in the first one instead?